### ISSUE TITLE
Staging Review finding: Avoid linking to PDFs whenever possible

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#105798]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** EDM and VEBT team
**Product:** GI Bill Comparison Tool
**Feature:** GI Bill Comparison Tool homepage

## Findings details
**VA.gov Experience Standard - issue:** [User experiences predictable interactions with components and patterns.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility), @erinrwhite (IA)

### Description

The user guide linked to in the footer is a 53-page PDF.

Even the best and most accessible PDFs can pose a potential barrier for users. Most notably:
- PDFs are not a good experience on mobile devices
- PDFs have a [bad reputation among screen reader users](https://webaim.org/projects/screenreadersurvey8/#pdf), and many screen reader users will give up on a process that requires them to dive deep into a PDF.

Even if the PDF is fully accessible (which I don't believe it is), I don't think it's reasonable to provide a 53-page PDF as the way to access this information.

## Recommended action

Is the user guide necessary? If the interface isn't intuitive on its own, providing a giant document that no one will ever read doesn't seem like the right solution to the problem.

If the user guide is necessary, then please provide this information as standard, HTML web content.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jkdrahota** [comment]
This update has been made.
